### PR TITLE
Handle unreachable git repos & page builder updates

### PR DESCRIPTION
**Pull request by rart**

### CODE REVIEW

File: ui/app/src/translations/locales/en.json [hunk 1]
```diff
@@ -295,10 +295,10 @@
   "publishingDashboard.confirmAllHelper": "Set the state for all selected items to \"Cancelled\"?",
   "publishingDashboard.confirmHelper": "Set the state for all selected items to \"Cancelled\"",
   "publishingDashboard.confirmHelperText": "Set item state to \"Cancelled\"?",
-  "publishingDashboard.environment": "Publishing Target",
```
Comment (sumerjabri) [inline ui/app/src/translations/locales/en.json]: Why?

File: ui/app/src/translations/locales/en.json [hunk 2]
```diff
@@ -295,10 +295,10 @@
...
+  "publishingDashboard.environment": "Publishing",
   "publishingDashboard.fetchPackagesFiles": "Fetch Packages Files",
   "publishingDashboard.filesList": "files list",
-  "publishingDashboard.filteredBy": "Showing: {state, select, all {} other {Status: {state}.}} {environment, select, all {} other {{environment} target.}} {path, select, none {} other {Filtered by {path}}}",
```
Comment (sumerjabri) [inline ui/app/src/translations/locales/en.json]: Why?

File: ui/app/src/translations/locales/en.json [hunk 3]
```diff
@@ -295,10 +295,10 @@
...
+  "publishingDashboard.filteredBy": "Showing: {state, select, all {} other {Status : {state}.}} {environment, select, all {} other {{environment} environment.}} {path, select, none {}  other {Filtered by {path}}}",
```
Comment (sumerjabri) [inline ui/app/src/translations/locales/en.json]: Why?

File: ui/app/src/translations/locales/en.json [hunk 4]
```diff
@@ -310,16 +310,18 @@
   "publishingDashboard.scheduled": "Scheduled for <b>{schedule, date, medium} {schedule, time, short}</b> by <b>{approver}</b>",
   "publishingDashboard.selectAll": "Select all on this page",
   "publishingDashboard.state": "State",
-  "publishingDashboard.status": "Status is {state} for {environment} publishing target",
+  "publishingDashboard.status": "Status is {state} for {environment} environment",
```
Comment (sumerjabri) [inline ui/app/src/translations/locales/en.json]: Why?

File: ui/guest/src/store/epics/root.ts [hunk 5]
```diff
@@ -396,7 +403,7 @@ const epic: Epic<GuestStandardAction, GuestStandardAction, GuestState> = combine
   },
   // endregion
 
-  // region trashDrop
+  // region TRASHED
```
Comment (sumerjabri) [inline ui/guest/src/store/epics/root.ts:406]: Is this intentional?

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: ui/guest/src/store/epics/root.ts (hunk 5 above)
Comment (rart) [reply ui/guest/src/store/epics/root.ts:406]: Yes.

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**